app/main.py

```diff
@@ -1,6 +1,15 @@
 from typing import Callable
+_CACHE = {}
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def wrapper(*args: int) -> int:
```

Comment:
_And yes, we have 2 functions to decorate, and both functions return int_
We are writing not a decorator for tests, when you write a function you should assume that you work with some sort of data, in our case what type of data can be cached?

Reply:
I don't understand what you want to tell me